system_tests/monitoring.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest2
+
+from gcloud import _helpers
+from gcloud.environment_vars import TESTS_PROJECT
+from gcloud import monitoring
+
+
+METRIC_TYPE = 'pubsub.googleapis.com/topic/send_message_operation_count'
+METRIC_KIND = monitoring.MetricKind.DELTA
+VALUE_TYPE = monitoring.ValueType.INT64
+
+RESOURCE_TYPE = 'pubsub_topic'
+
+
+def setUpModule():
+    _helpers.PROJECT = TESTS_PROJECT
+
+
+class TestMonitoring(unittest2.TestCase):
```

Comment:
I see from your comment below that the goal of the tests is to just verify the queries on whatever existing state is in the project, yes?